ci_framework/roles/devscripts/tasks/sub_tasks/16_user.yml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Can we add a cleanup task for users? It will be helpful to keep the environment clean if a single machine is used for multiple time deployment in CI.